db/schema.rb

```diff
@@ -27,10 +36,21 @@
   end
 
   create_table "purchases", force: :cascade do |t|
-    t.integer  "user_id",    null: false
-    t.integer  "product_id", null: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.integer  "user_id",             null: false
+    t.integer  "product_id",          null: false
+    t.datetime "created_at",          null: false
+    t.datetime "updated_at",          null: false
+    t.integer  "purchasing_quantity"
```

Comment:
Things so strange. I don't have the migration in current branch, but I do have the columns...